server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -265,10 +266,21 @@ public static UserRepresentation toBriefRepresentation(UserModel user) {
         rep.setEnabled(user.isEnabled());
         rep.setEmailVerified(user.isEmailVerified());
         rep.setFederationLink(user.getFederationLink());
+        addAttributeToBriefRep(user, rep, IS_TEMP_ADMIN_ATTR_NAME);
 
         return rep;
     }
 
+    private static void addAttributeToBriefRep(UserModel user, UserRepresentation userRep, String attributeName) {
+        String userAttributeValue = user.getFirstAttribute(attributeName);
+        if (userAttributeValue != null) {
```

Comment:
```suggestion
        if (StringUtil.isNotBlank(userAttributeValue)) {
```

Not sure whether we accept empty attributes - but if there's some existing specified like that, it can be propagated to the brief rep IMHO. Thus, LGTM.

cc @vmuzikar

Reply:
@mabartos I wouldn't do that. Exactly as you mentioned, the attr is already there and there may be a reason why it's empty. Here we're just copy-pasting what we already have in the model.

Reply:
ok, agree